data_model/src/account.rs

```diff
@@ -124,9 +125,19 @@ impl Default for SignatureCheckCondition {
 /// Builder which should be submitted in a transaction to create a new [`Account`]
 #[allow(clippy::multiple_inherent_impl)]
 #[derive(
-    Debug, Display, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema,
+    Debug,
+    Display,
+    Clone,
+    OrdEqHash,
```

Comment:
Exactly

Reply:
Also, I think that there are only two possible implementations; comapre by id vs. compare by all fields. So this could always be derived. Not sure if that is a good idea

Reply:
> Don't quite understand what you mean by that. Are you saying that instead of having a separate `OrdEqHash` derive, we could put it into `Identifiable` derivation?

My question is also. Why don't we always derive `Ord`?